extensions/mybatis/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: "MyBatis SQL Mapper"
+metadata:
+  keywords:
+  - "mybatis"
+  guide: "https://quarkus.io/guides/mybatis"
```

Comment:
I don't see a documentation page for this link. You need to add a `mybatis.adoc` in `docs/src/main/asciidoc`